shared/api/network.go

```diff
@@ -152,6 +152,12 @@ type NetworkLease struct {
 	//
 	// API extension: network_leases_location
 	Location string `json:"location" yaml:"location"`
+
+	// Name of the project of the entity related to the hostname
+	// Example: default
+	//
+	// API extension: network_allocations_ovn_uplink
+	HostProject string `json:"project" yaml:"project"`
```

Comment:
Why HostProject and not the more usual Project?

Reply:
My thinking was to better convey that the project relates to the host, not the network nor the lease itself. If it is too redundant just let me know.

Reply:
Precisely, I couldn't think of a better name to convey this. Please feel free to make suggestions :)

Reply:
Lets just use `Project` like we do everywhere else